extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -93,13 +106,28 @@ public void initialize() {
                 updateStatus(statusInfo.getStatus(), statusInfo.getStatusDetail(), statusInfo.getDescription());
                 FullLight fullLight = getLight();
                 if (fullLight != null) {
+                    String modelId = fullLight.getModelID().replaceAll(NORMALIZE_ID_REGEX, "_");
+                    updateProperty(Thing.PROPERTY_MODEL_ID, modelId);
```

Comment:
Yes, it is...

> Yes, it is already a bug that there is another such call (introduced by the firmware stuff). So far it was forbidden for bindings to add properties themselves.

But I dont get what the firmware stuff has additionally introduced here. Or do you mean that there is a dependency on the firmware version property now that has to be set by the handler (Most probabyl during intialize)?

And why was it forbidden? Because of the technical problem we have with the updateProperties call?